web/src/containers/Sidebar.js

```diff
@@ -1,219 +1,215 @@
-    <aside className="site-sidebar bg-teal ">
-      <div className="xs-hide sm-hide">
-        <div className="px2 py3 lg-px3 lg-py4 bg-white flex items-center">
-          <img
-            src={`/static/img/states/${place.id}.svg`}
-            alt={place.name}
-            className="align-middle mr2"
-            width="40"
-            height="40"
-          />
-          <h1 className="m0 blue h3 light caps line-height-2">
-            {place.name} <br />
-            {t('title', { year: '2018' })}
-          </h1>
-        </div>
-        <div className="p2 lg-p3">
-          <ul className="list-reset">
-            {linkGroup1.map(d => (
-              <SidebarLink
-                key={d.id}
-                anchor={d.id}
-                expanded={expanded[d.id]}
-                hash={hash}
-                sub={d.sub}
-                toggleExpand={() => toggleExpand(d.id)}
-              >
-                {d.name}
-              </SidebarLink>
-            ))}
-            {expanded.activities && (
-              <ul className="mb0 list-reset">
-                {activities.map((a, i) => (
-                  <SidebarLink
-                    key={a.key}
-                    anchor={a.anchor}
-                    depth={1}
-                    hash={hash}
-                    onClick={() => expandSection(a.key)}
-                  >
-                    {t(`sidebar.titles.activity-${a.name ? 'set' : 'unset'}`, {
-                      number: i + 1,
-                      name: a.name
-                    })}
-                  </SidebarLink>
-                ))}
-              </ul>
-            )}
-            {linkGroup2.map(d => (
-              <SidebarLink
-                key={d.id}
-                anchor={d.id}
-                expanded={expanded[d.id]}
-                hash={hash}
-                sub={d.sub}
-                toggleExpand={() => toggleExpand(d.id)}
-              >
-                {d.name}
-              </SidebarLink>
-            ))}
-          </ul>
-          <div className="mt3">
-            <Btn onClick={() => saveApdToAPI()}>
-              {t('sidebar.saveApdButtonText')}
-            </Btn>{' '}
-            <Btn kind="outline" extraCss="bg-white blue">
-              {t('sidebar.savePdfButtonText')}
-            </Btn>
+    return (
+      <div className="ds-l-col--3 bg-white">
+        <aside className="site-sidebar">
+          <div className="xs-hide sm-hide">
+            <div className="flex items-center">
+              <img
+                src={`/static/img/states/${place.id}.svg`}
+                alt={place.name}
+                className="align-middle mr2"
+                width="40"
+                height="40"
+              />
+              <h1 className="text-xl">
+                {place.name} <br />
+                {t('title', { year: '2018' })}
+              </h1>
+            </div>
+            <VerticalNav
+              selectedId={this.props.selectedId}
```

Comment:
@mgwalker I think I need a pair to try to figure out what I'm doing wrong here 😞